src/System.Windows.Forms/tests/AccessibilityTests/Main.Designer.cs

```diff
@@ -129,11 +130,22 @@ private void InitializeComponent()
             this.btnDataBindingExample.UseVisualStyleBackColor = true;
             this.btnDataBindingExample.Click += new System.EventHandler(this.btnDataBindingExample_Click);
             // 
+            // BtnCusAccName
+            // 
+            this.btnCusAccName.Location = new System.Drawing.Point(12, 410);
+            this.btnCusAccName.Name = "cusAccName";
+            this.btnCusAccName.Size = new System.Drawing.Size(455, 23);
+            this.btnCusAccName.TabIndex = 9;
+            this.btnCusAccName.Text = "Testing controls whose accessible name, role, description have been customized";
```

Comment:
To make it a little more consistent with the rest:
```suggestion
            this.btnCusAccName.Text = "Accessible properties: Testing controls with custom accessible name, role, and description";
```